subjects/localhost/audit/README.md

```diff
@@ -0,0 +1,67 @@
+#### Functional
+
+#### Localhost is about program your own HTTP server and test it with an actual browser.
+###### Take The necessary time to understand the project and to test it,take a look around the code to ask reasonable questions such as:
```

Comment:
- missing space after ",".
- "take a  look around the code **and** ask ...", "and" instead of "to".